qiskit_experiments/framework/backend_data.py

```diff
@@ -24,6 +25,9 @@ class BackendData:
 
     def __init__(self, backend):
         """Inits the backend and verifies version"""
+        warnings.filterwarnings(
```

Comment:
Hmm, I am not sure which context you are thinking of here -- just Qiskit Experiments or something more broadly about pulse and qobj? This specific filter was added in #1470 and I commented on it there.

On the user-facing side of Qiskit Expeirments, I don't think we need to do much besides removing the things deprecated in this PR. I would say we don't need to do anything, but it is hard to be sure until we do the removals and see if there is still anything referencing pulse that we missed. Right now the tests generate so many warnings about Pulse deprecation that it is hard to trace them.

On the developer side of Qiskit Experiments, we may have to fix some tests if they need to be kept because they are not testing pulse features but are using pulse-related backends that are deprecated.

I was not sure though if you were asking about migrating pulse qobj more generally for pulse support like the planned continuation of pulse support in Qiskit Dynamics?

Reply:
I don't think Qiskit Dynamics requires wire formant so deprecation of qobj should be generally fine. I was just wondering if we have any component relying on qobj. If you add this filter just to make the log clean then it should be fine. We should remember to remove the filter after the runtime removes all backends with pulse information with removal of the qobj module in the qiskit core -- because the warning filter add some overhead.

Reply:
Yes, when we process the deprecations, we should remove all filters and hopefully not see warnings about Qobj.